src/main/scala/Column.scala

```diff
@@ -1181,24 +1233,24 @@ case class Column private (
 
   def validateHeader(columnName: String): WarningsAndErrors = {
     var errors = Array[ErrorWithCsvContext]()
-    if (titleValues.nonEmpty) {
-      var validHeaders = Array[String]()
-      for (title <- titleValues) {
-        if (Column.languagesMatch(title, lang)) {
-          validHeaders :+= title
-        }
-      }
-      if (!validHeaders.contains(columnName)) {
-        errors :+= ErrorWithCsvContext(
-          "Invalid Header",
-          "Schema",
-          "1",
-          columnOrdinal.toString,
-          columnName,
-          titleValues.mkString(",")
-        )
+//    if (titleValues.nonEmpty) {
```

Comment:
Can we remove the commended out if statement here and below now?

Reply:
Sure, missed it